### PR TITLE
URI-encode-on-open-html-file

### PR DESCRIPTION
**Pull request by aUsefulMethod**
I'm able to open .html files with # in name when opening with responsively. I've tested a few other reserved characters and it seems to be working as well.

### CODE REVIEW

File: desktop-app/app/main.dev.js [hunk 1]
```diff
@@ -91,7 +91,10 @@ const openWithHandler = filePath => {
     filePath != null &&
     (filePath.endsWith('.html') || filePath.endsWith('.htm'))
   ) {
-    const url = `file://${filePath}`;
+    // code to encode
+    // file URI
+    p1 = encodeURIComponent(filePath);
+    const url = `file://${filePath + p1}`;
```
Comment (manojVivek) [inline desktop-app/app/main.dev.js:97]: @aUsefulMethod Isn't `filePath + p1` going to repeat the path twice?

**Comment by manojVivek** [comment]
@aUsefulMethod Can you please attach a screenshot of how the address bar looks after the encoding?

**Comment by jzabala** [comment]
Thanks for the PR @aUsefulMethod 🙂

I have a question about what problem is this PR solving? 

From what I see Chromium is already encoding the path.

Aug-20-2020 11-26-47

File: desktop-app/app/main.dev.js (hunk 1 above)
Comment (mrfelfel) [reply desktop-app/app/main.dev.js:97]: maybe not work ... encodeURIComponent  encode all of string :D

**Comment by aUsefulMethod** [comment]
I’m currently in school, I’ll close this PR when I get a chance.

**Comment by jjavierdguezas** [comment]
Hi @aUsefulMethod we are going with #415, we think that is a better approach. Thanks anyway and we are looking forward to more collaboration in the future! 🙌

**Pull request closed without merging**